examples/server/tests/unit/test_completion.py

```diff
@@ -85,7 +85,8 @@ def test_completion_stream_vs_non_stream():
     assert content_stream == res_non_stream.body["content"]
 
 
-@pytest.mark.parametrize("n_slots", [1, 2])
+#@pytest.mark.parametrize("n_slots", [1, 2])
+@pytest.mark.parametrize("n_slots", [1])
```

Comment:
Does it work if here we set temperature to 0.0 instead of forcing it to 1 slot?

Reply:
```py
res = server.make_request("POST", "/completion", data={
            "prompt": "I believe the meaning of life is",
            "seed": 42,
            "temperature": 1.0, # <==== Change to 0.0
            "cache_prompt": False,  # TODO: remove this once test_cache_vs_nocache_prompt is fixed
})
```

Reply:
I don't know. I'll have a try.

Reply:
Yes it look to work with 2 slot and temperature = 0.0